src/community/security/oauth2/oauth2-web/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We have the module correctly separated from the core one but is not added as a dependency where it is needed.